Source/WebCore/rendering/svg/SVGInlineTextBox.cpp

```diff
@@ -52,14 +53,15 @@ struct ExpectedSVGInlineTextBoxSize : public LegacyInlineTextBox {
     float float1;
     uint32_t bitfields : 5;
     void* pointer;
+    SVGPaintServerOrColor paintServerOrColor;
```

Comment:
Does this field need if ENABLE(LAYER_BASED_SVG_ENGINE) ?

Reply:
Good catch!